[DOCFIX] Improve UFS ozone doc (#15462)
Alluxio/alluxio: docs/en/ufs/Ozone.md

or `o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>/alluxio/data` if only the directory `/alluxio/data` inside the ozone bucket `<OZONE_BUCKET>` of `<OZONE_VOLUME>` is mapped to Alluxio.

set the property `alluxio.master.mount.table.root.option.alluxio.underfs.hdfs.configuration` in `conf/alluxio-site.properties` to point to your `ozone-site.xml`. Make sure this configuration is set on all servers running Alluxio.

```properties
alluxio.master.mount.table.root.ufs=o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>/
alluxio.master.mount.table.root.option.alluxio.underfs.hdfs.configuration=/path/to/hdfs/conf/ozone-site.xml
``` 

## Ozone HA Mode

To make Alluxio mount Ozone in HA mode, you should configure Alluxio's server so that it can find the OzoneManager. Please note that once set up, your application using the Alluxio client does not require any special configuration.
In HA mode `alluxio.master.mount.table.root.ufs` needs to specify `<OM_SERVICE_IDS>`
such as:
```properties
alluxio.master.mount.table.root.ufs=o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>.<OM_SERVICE_IDS>/
alluxio.master.mount.table.root.option.alluxio.underfs.hdfs.configuration=/path/to/hdfs/conf/ozone-site.xml
``` 

> `<OM_SERVICE_IDS>` can be found in `ozone-site.xml`
> Possible `ozone-site.xml` file, as known from the file, `<OM_SERVICE_IDS>` is `omservice1`

Reviewer: ```suggestion
`<OM_SERVICE_IDS>` can be found in `ozone-site.xml`.
In the following example `ozone-site.xml` file, `<OM_SERVICE_IDS>` is `omservice1`:
```

Author: done

```xml
<property>
    <name>ozone.om.service.ids</name>
    <value>omservice1</value>
</property>
```

### Mount HDFS with Specific Versions

> User can mount an Ozone cluster with a specified version as an under storage into Alluxio namespace.

Reviewer: ```suggestion
Users can mount an Ozone cluster of a specific version as an under storage into Alluxio namespace.
```

Author: done

set the property `alluxio.master.mount.table.root.option.alluxio.underfs.hdfs.configuration` in `conf/alluxio-site.properties` to point to your `ozone-site.xml`. Make sure this configuration is set on all servers running Alluxio.

```
alluxio.master.mount.table.root.ufs=o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>/
alluxio.master.mount.table.root.option.alluxio.underfs.hdfs.configuration=/path/to/hdfs/conf/ozone-site.xml
``` 

## Ozone HA Mode

To configure Alluxio's OzoneManager for Ozone in HA mode, you should configure Alluxio's server to access Ozone. Please note that once set up, your application using the Alluxio client does not require any special configuration.
In HA mode `alluxio.master.mount.table.root.ufs` needs to specify `<OM_SERVICE_IDS>`

```
alluxio.master.mount.table.root.ufs=o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>.<OM_SERVICE_IDS>/
alluxio.master.mount.table.root.option.alluxio.underfs.hdfs.configuration=/path/to/hdfs/conf/ozone-site.xml
``` 

`<OM_SERVICE_IDS>` can be found in `ozone-site.xml`
Possible `ozone-site.xml` file, as known from the file, `<OM_SERVICE_IDS>` is `tdw`
```xml
<property>
    <name>ozone.om.service.ids</name>

>     <value>tdw</value>

Author: done

  /ozone o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>/
```

> If you need mount an HDFS cluster with a specified version, you can specify it through `alluxio.underfs.version=<OZONE_VERSION>`.

Reviewer: ```suggestion
If you need to mount an Ozone cluster of a specific version, you can specify it through the command line option `alluxio.underfs.version=<OZONE_VERSION>`.
```

Author: done
